src/main/java/sentinel/command/Command.java

```diff
@@ -44,6 +44,7 @@ public Command(Ui ui, SentinelList list) {
     public static Command createCommand(Sentinel.CommandType commandType, Ui ui, SentinelList list) {
         return switch (commandType) {
             case list -> new ListCommand(ui, list);
+            case find -> new FindCommand(ui, list);
```

Comment:
I think its your coding of switch cases is good but the coding standards specify the following format instead 🤔

``` java
switch (condition) {
case ABC:
    statements;
    // Fallthrough
case DEF:
    statements;
    break;
case XYZ:
    statements;
    break;
default:
    statements;
    break;
}



```

Reply:
Thank you for highlighting this! I'd changed it on a more recent commit after someone brought it up on the forums but haven't opened a PR yet. Thank you for reminding me!